packages/validation-manager/src/ValidationManagerRIP7560.ts

```diff
@@ -11,6 +10,7 @@ import {
 import { IValidationManager, ValidateUserOpResult, ValidationResult } from './IValidationManager'
 import { eth_traceRip7560Validation } from './GethTracer'
 import { tracerResultParser } from './TracerResultParser'
+import { AA_ENTRY_POINT } from '@account-abstraction/bundler/src/BundlerConfig'
```

Comment:
Let's export it from the `index.ts` file to avoid this deep linking?

Reply:
or better, have "constants" file

Reply:
So.. which one is it?

Reply:
I moved AA_ENTRY_POINT to validation manager since it's only used there. If we ever need it elsewhere, we can add a constants file and use it. Until then...